src/annotations/AnnotationParser.php

```diff
@@ -13,6 +13,10 @@
 
 namespace mindplay\annotations;
 
+if (!defined('T_TRAIT')) {
+    define('mindplay\\annotations\\T_TRAIT', -2);
```

Comment:
To avoid polluting the global namespace. What if another library is attempting to detect trait support by checking `defined('T_TRAIT')` but needs a fallback other than -2?